syntax = "proto3";

package requester;

service Matcher{
	rpc Request(WhichUser) returns (UserList){}
}

message WhichUser{
	string serverID = 1;
}
message UserList {
	repeated string userid = 1;
	repeated string username = 2;
}

